projects/erigon_focil.md

```diff
@@ -0,0 +1,94 @@
+# Fork-Choice Enforced Inclusion Lists (FOCIL) For Erigon 
+
+## Motivation
+
+[EIP-7805](https://eips.ethereum.org/EIPS/eip-7805) is a potential candidate for the glamsterdame upgrade which focuses on improving the block building on Ethereum. Currently block building is centralized and not resistant to censorship. With this EIP ethereum expects to fix these issues and improve the block building capabilities of the chain.
```

Comment:
```suggestion
[EIP-7805](https://eips.ethereum.org/EIPS/eip-7805) is a potential candidate for the Glamsterdam upgrade which focuses on improving the block building on Ethereum. Currently block building is centralized and not resistant to censorship. With this EIP, Ethereum expects to fix these issues and improve the block building capabilities of the chain.
```
> improve the block building capabilities of the chain.

Just a detail but block building capabilities is not really what I would consider the improvement, just being able to create commitment of external block builders in the protocol